packages/typespec-client-generator-core/test/examples/load.test.ts

```diff
@@ -403,3 +403,69 @@ it("teamplate case", async () => {
   strictEqual(operation.examples?.length, 1);
   strictEqual(operation.examples![0].filePath, "template.json");
 });
+
+it("multiple services without versioning", async () => {
```

Comment:
It is better to add more cases, including positive and negative cases to verify the behavior.

Reply:
Added.